script.js

```diff
@@ -0,0 +1,20 @@
+const form = document.getElementById("my-form");
+
+async function handleSubmit(event) {
+  let data = new FormData(event.target);
+  fetch(event.target.action, {
+    method: form.method,
+    body: data,
+    headers: {
+      Accept: "application/json",
+    },
+  })
+    .then((response) => {
+      console.log("success");
+      form.reset();
+    })
+    .catch((error) => {
+      console.log("error");
+    });
+}
+form.addEventListener("submit", handleSubmit);
```

Comment:
- [ ] Kindly delete this file, one of the major requirements for the project is that you do not implement any functionality using **_JavaScript_**. 👍